docs/source/viewer.rst

```diff
@@ -29,8 +30,101 @@ button to add a new server. Here are two demo servers that you may add for testi
 Functionality
 =============
 
-Coming soon...
+The xcube viewer functionality is described exemplary using the `DCS4COP Demo viewer`_.
+The xcube viewer includes a basemap layer with visualized data retrieved from the data cubes on top.
+One can zoom in and out by using the zooming buttons in the top right corner of the map window or using
+the zoom function of the computer mouse. A scale for the map is located in the lower right corner and in the
+upper left corner a corresponding legend to the mapped data of the data cube is available.
 
+.. image:: _static/screenshot_viewer_docu_1.png
+  :width: 800
+
+A xcube viewer may hold several xcube datasets which can be selected via the drop-down menu `Dataset`.
+The viewer automatically adjusts the viewed area to the selected xcube dataset, meaning that if a newly selected
+dataset is located in a different region, the map will display the correct region.
```

Comment:
Change from active to passive voice.

Reply:
Argh, I was not clear! By "Change from active to passive voice" I meant, I recognized that you use passive voice, while other older sections use active voice.